MetaCell/geppetto-netpyne
# parent section in topology should be SelectField

this requires: https://github.com/MetaCell/NetPyNE-UI/pull/53

The parent section to connect a given section should be a selectField with the already defined sections within the cellRule. With this, we are calling a netpyne_geppetto method to pull all the available sections, that should not be the case (section int should be restrained to the sections belonging to the current cellRule). 

Openning a Trello card to pick up from here

### to Test:
- Go to CellRule, 
- Create a CellRule, 
- Go to Section,
- Create a bunch of sections,
- Go to tab topology
- Click parent Section.... Now it should not be a textField but a selectField showing the name of already defined sections
